sensu-plugins-aws.gemspec

```diff
@@ -38,7 +38,7 @@ Gem::Specification.new do |s|
   # https://github.com/fog/fog-core/issues/206
   s.add_runtime_dependency 'fog-core',          '1.43.0'
   s.add_runtime_dependency 'right_aws',         '3.1.0'
-  s.add_runtime_dependency 'sensu-plugin',      '~> 1.3'
+  s.add_runtime_dependency 'sensu-plugin',      '>= 1.3'
```

Comment:
that's rather dangerous as this no longer locks to major versions. I think what you want is `~> 2.0` and to make this a breaking change.